Mathlib/Algebra/Group/Action/Pointwise/Finset.lean

```diff
@@ -78,7 +78,7 @@ instance isCentralScalar [SMul α β] [SMul αᵐᵒᵖ β] [IsCentralScalar α
 @[to_additive
       "An additive action of an additive monoid `α` on a type `β` gives an additive action
       of `Finset α` on `Finset β`"]
-protected def mulAction [DecidableEq α] [Monoid α] [MulAction α β] :
+protected noncomputable def mulAction [DecidableEq α] [Monoid α] [MulAction α β] :
     MulAction (Finset α) (Finset β) where
```

Comment:
Sorry what! Why is this becoming non-computable?

Reply:
Because it depends, transitively, on `Set.semigroup`, which has always been noncomputable.

One could (please do!) modify `Finset.monoid` to not use `Set.monoid`.

Reply:
Yes, see [#nightly-testing > #23806 adaptations for nightly-2025-04-07 @ 💬](https://leanprover.zulipchat.com/#narrow/channel/428973-nightly-testing/topic/.2323806.20adaptations.20for.20nightly-2025-04-07/near/510836584)

Reply:
I've read that, but at no point does it talk about using noncomputable declarations in proofs